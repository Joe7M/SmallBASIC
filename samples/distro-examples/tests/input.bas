v = "111111111111111111111111111111111111111111111111111111111111111111111111111"
sinput v; k
print k
