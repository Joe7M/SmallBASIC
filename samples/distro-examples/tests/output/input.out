1.11111111111111E+74
